tooling/commands/test.js

```diff
@@ -156,7 +156,9 @@ module.exports = {
 
         /** Each package is run through testPackage util */
         for (const packageName of argv.packages) {
-          await testPackage(argv, packageName);
+          const onMocha = packageName === '@webex/plugin-meetings' || packageName === 'webex';
```

Comment:
Hope this does not grows , But if it does lets move the names to a array constant

Reply:
Yeah, for now jest is running good on other packages than these two.